docs/api-reference/api-introduction.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: API Reference
+sidebar_label: Introduction
+---
+
+View complete documentation of our GraphQL API.
```

Comment:
I think this file is unused?

Reply:
Removed reference landing page from main directory (docs/api-reference), modified script and now its place is in "docs/api-reference/api-introduction.mdx". Also file contains now date of generation.